WBooks/Debug copy.xcconfig

```diff
@@ -0,0 +1,11 @@
+//
+//  Debug.xcconfig
+//  WBooks
+//
+//  Created by Juan Silva on 03/06/2021.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+EXAMPLE = example_value
```

Comment:
Let's delete these files: `Debug copy.xcconfig` and `Development copy.xcconfig`. They are likely not appearing in the project structure, but it you search them with Finder or the Terminal they will show up.